arch/arm/core/aarch32/cortex_a_r/Kconfig

```diff
@@ -111,6 +118,19 @@ config ARMV7_R_FP
 	  This option signifies the use of an ARMv7-R processor
 	  implementation supporting the Floating-Point Extension.
 
+config ARMV8_R
```

Comment:
Issue there is that ARMV8_R is already defined in
arch/arm64/core/Kconfig

Any suggestion, does we should name it differently ?

Reply:
`AARCH32_ARMV8_R` ?